tests/integration/synapseutils/test_synapseutils_sync.py

```diff
@@ -34,9 +35,9 @@ def __init__(self):
             self.f2 = utils.make_bogus_data_file(n=10)
             self.f3 = "https://www.synapse.org"
 
-            self.header = "path	parent	used	executed	activityName	synapseStore	foo	date_1	datetime_1	datetime_2	datetime_3\n"
+            self.header = "path	parent	used	executed	activityName	synapseStore	foo	date_1	datetime_1	datetime_2	datetime_3	multiple_strings	multiple_dates	multiple_bools	multiple_ints	multiple_floats\n"
             self.row1 = (
-                '%s	%s	%s	"%s;https://www.example.com"	provName		bar	2020-01-01	2023-12-04T07:00:00Z	2023-12-05 23:37:02.995000+00:00	2023-12-05 07:00:00+00:00\n'
+                '%s	%s	%s	"%s;https://www.example.com"	provName		bar	2020-01-01	2023-12-04T07:00:00Z	2023-12-05 23:37:02+00:00	2023-12-05 07:00:00+00:00	a,b,c,d	2020-01-01,2023-12-04T07:00:00.111Z,2023-12-05 23:37:02.333+00:00,2023-12-05 07:00:00+00:00	fAlSe,False,tRuE,True	1,2,3,4	1.2,3.4,5.6,7.8\n'
```

Comment:
Do we have a test case of a string with escaped commas?  What does that look like right now when theres a table value with 'my sentence, comma, has commas'?

Reply:
I added this.

This is what this looks like in google sheets for this situation:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/e02bfaf2-e911-497c-92d9-cb3abe24f77f)